src/lib.rs

```diff
@@ -74,6 +74,8 @@ static MSG_ARG: &str = "msg";
 static PARAMS_ARG: &str = "params";
 static GAS_USED_ARG: &str = "gas_used";
 
+const PRINT_DEBUG: bool = false;
```

Comment:
Good one.
I guess we could temporarily turn this on if we needed for a test?

Reply:
Yes that would work. I created a variable primarily to have a name instead of a magic `, false)` argument you never know what it is.